app/assets/javascripts/actions/profile_actions.js

```diff
@@ -0,0 +1,13 @@
+import McFly from 'mcfly';
+const Flux = new McFly();
+import API from '../utils/api.js';
+
+const ProfileActions = Flux.createActions({
+  fetch_stats(username) {
+    return API.fetchStatsData(username)
```

Comment:
This is not a specific enough function name in `API`. It needs to clearly indicate that it's fetching user profile stats.